src/main/java/frc/team2412/robot/subsystems/ArmSubsystem.java

```diff
@@ -126,42 +113,50 @@ public static class ArmConstants {
 		 * Values are angles that correspond to specific arm positions:
 		 * Arm Angle
 		 * Retracted Wrist Angle
-		 * Retracted Wrist Angle (cone edition)
 		 * Prescore Wrist Angle
 		 * Scoring Wrist Angle
+		 *
 		 */
 		public static enum PositionType {
-			UNKNOWN_POSITION(0.212, 0.122, 0.122, 0.477, 0.477),
-			ARM_LOW_POSITION(0.212, 0.122, 0.122, 0.477, 0.477),
-			ARM_MIDDLE_POSITION(0.415, 0.122, 0.122, 0.459, 0.521),
-			ARM_HIGH_POSITION(0.6546, 0.122, 0.122, 0.473, 0.52),
-			ARM_SUBSTATION_POSITION(0.567, 0.122, 0.122, 0.57, 0.57); // ?
+			UNKNOWN_POSITION(0.212, 0.122, 0.477, 0.477, -1),
+			ARM_CARRY_POSITION(0.212, 0.212, 0.122, 0.122, -1),
+
+			ARM_LOW_CUBE_POSITION(0.212, 0.122, 0.122, 0.477, -1),
+			ARM_MIDDLE_CUBE_POSITION(0.415, 0.122, 0.459, 0.521, -1),
+			ARM_HIGH_CUBE_POSITION(0.6546, 0.122, 0.473, 0.52, -1),
+			ARM_SUBSTATION_CUBE_POSITION(0.567, 0.122, 0.57, 0.57, -1),
+
+			ARM_LOW_CONE_POSITION(0.212, 0.122, 0.122, 0.477, 1),
+			ARM_LOW_CONE_HORIZONTAL_POSITION(0.212, 0.122, 0.122, 0.477, 1),
+			ARM_MIDDLE_CONE_POSITION(0.415, 0.122, 0.459, 0.521, 1),
+			ARM_HIGH_CONE_POSITION(0.6546, 0.122, 0.473, 0.52, 1),
+			ARM_SUBSTATION_CONE_POSITION(0.567, 0.122, 0.57, 0.57, 1);
 
 			public final double armAngle;
 			public final double retractedWristAngle;
-			public final double retractedConeWristAngle;
-
 			public final double prescoringWristAngle;
 			public final double scoringWristAngle;
+			public final int rollerDirection;
 
 			PositionType(
 					double armAngle,
 					double retractedWristAngle,
-					double retractedConeWristAngle,
 					double prescoringWristAngle,
-					double scoringWristAngle) {
+					double scoringWristAngle,
+					int rollerDirection) {
```

Comment:
The rollerDirection is being used so that we can pull a multiplier for the way the intake spins from the positionType enum since iirc the ultimate goal is to have the arm and write positions stored in one type